docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2157,53 +2157,127 @@ limitation.
+  is likely missing key real-life challenges.
+
+We consider all simulants age 18 and over to be working-age; all such
+simulants either have an employer or are considered unemployed.
+We only allow a single employer at a time for each simulant.
+
+Each employer has a single street address.
+
+Initialization
+^^^^^^^^^^^^^^
+
+We initialize 58% of the working-age population to be unemployed,
+regardless of demographics.
+We initialize 3% of the working-age population to be employed by the military,
+regardless of demographics.
+We consider the military to be a single employer, which has the same street address
+as the military group quarters "household."
+
+.. todo::
+
+  Source or update these numbers.
+
+We generate (non-military) employers with an initial size attribute chosen from a skewed distribution to
+ensure that there are a few large employers and many small employers.
+Specifically, we use a log-normal distribution with :math:`\mu=4` and :math:`\sigma=1`,
+which means that the mean employer has ~90 employees and the median has ~55 employees.
+Each one has a street address, generated in the same way that residential
+addresses are generated, but without any restriction on state/PUMA/ZIP.
+
+.. todo::
+
+  This business size distribution doesn't seem plausible!
+
+.. todo::
+
+  Document how we name businesses.
+
+We generate the number of employers needed to (in expectation) employ our starting
+non-military employed population, that is:
+
+.. math::
+
+  \text{num_employers}=
+  \frac{\text{num_simulants}}{E(\text{employer_size})}=
+  \frac{\text{num_simulants}}{e^{\mu_\text{size}+\frac{1}{2}*\sigma_\text{size}}}
+
+In order to give individual simulants employers such that the size attribute is (roughly)
+accurate at the population level, we select each simulant's employer from the categorical
+distribution where the probability of each employer is proportional to its size.
+
+Finally, we set all working-age simulants living in military group quarters to be employed by the
+military.
+This is in addition to the 3% assigned across the entire working-age population regardless of
+living situation.
+
+Updating employer over time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Working-age simulants (including those who are unemployed) change employment randomly
+at the rate of 50 changes per 100 person-years.
+This rate includes those who change from employed to unemployed.
+
+If a simulant selected for an employment change lives in military group quarters,
+no employment change occurs;
+they remain employed by the military.
+This means that in practice, the actual rate of employment changes will be a bit
+less than 50 per 100 person-years.
+
+When an employed simulant changes employment, they have a 58% chance of
+becoming unemployed and a 3% chance of becoming employed by the
+military.
+Otherwise, they sample a new non-military employer with probability
+proportional to employer size, as at initialization.
```

Comment:
note that this is specifically proportional to the employer sizes as they were at initialization. Over time this results in the employer sizes moving towards ... something less skewed than they were to start. I'm not sure precisely where it hits equilibrium